### PR TITLE
Add new sections to Methods and include commands for each tool.

### PR DESCRIPTION
**Pull request by janstett**
Note: @janstett will add references soon.

### CODE REVIEW

File: data.Rmd [hunk 1]
```diff
@@ -48,31 +48,130 @@ List of directories and their contents
...
+
+#### SAG Assembly Quality Control and Decontamination
+
+Quality of the SAGs was assessed using CheckM(v1.0.5) with the following command:
+
+```
+checkm lineage_wf --tab_table -x .fna --threads 8 --pplacer_threads 8
+```
+
+If a SAG was >5% contamination, then the SAGs were passed through ProDeGe v2.3.0[ref].The resulting decontaminated SAG was re-assesed with CheckM. If the SAG still exceeded 5% contamination after several iterations, then short contigs <2000bp were removed from the assembly and re-assessed with CheckM.
```
Comment (cmorganl) [inline data.Rmd:164]:
```suggestion
SAGs found to have >5% contamination were passed through ProDeGe v2.3.0[ref].The resulting decontaminated SAGs were re-assesed with CheckM and those that still exceeded 5% contamination had all of their short contigs (i.e. <2000 bp) removed from the assembly. These were again re-assessed with CheckM to ensure they met our standards. 
```

File: data.Rmd [hunk 2]
```diff
@@ -48,31 +48,130 @@ List of directories and their contents
...
+
+Quality of the SAGs was assessed using CheckM(v1.0.5) with the following command:
+
+```
+checkm lineage_wf --tab_table -x .fna --threads 8 --pplacer_threads 8
+```
+
+If a SAG was >5% contamination, then the SAGs were passed through ProDeGe v2.3.0[ref].The resulting decontaminated SAG was re-assesed with CheckM. If the SAG still exceeded 5% contamination after several iterations, then short contigs <2000bp were removed from the assembly and re-assessed with CheckM. 
+
+For this project, Based on the resulting CheckM scores, SAGs that were >50% Complete with <10% contamination were selected.
```
Comment (cmorganl) [inline data.Rmd:166]:
```suggestion
For this project, SAGs that were estimated by CheckM to have >50% completeness and <10% contamination were selected.
```

**Review by cmorganl — COMMENTED** [review]
Looks great overall! Just need to include the references before I merge it into `development`.

File: data.Rmd [hunk 3]
```diff
@@ -48,31 +48,130 @@ List of directories and their contents
...
 
 ## Methods
 
-Water was collected from various depths at Saanich Inlet and filtered through a 0.22 µm Sterivex filter to collect biomass. Total genomic DNA and RNA were extracted from these filters. RNA was reversed transcribed into cDNA and both genomic DNA and cDNA were used in the construction of shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with **2x150bp** technology. For more in-depth sampling and sequencing methods, please see @Hawley.2017.
+### Metagenomes and metatransriptopmes
 
-### Quality filtering of reads
+#### Sample Collection
+Water was collected at 10, 100, 120, 135, 150, 165, and 200m at Saanich Inlet on August 1st, 2012 and filtered through a 0.22 µm Sterivex filter to collect biomass. Total genomic DNA and RNA were extracted from these filters. RNA was reversed transcribed into cDNA and both genomic DNA and cDNA were used in the construction of shotgun Illumina libraries. Sequencing data were generated on the Illumina HiSeq platform with **2x150bp** technology. For more in-depth sampling and sequencing methods, please see @Hawley.2017.
```
[the remark this answers is not in the document]
Comment (janstett) [reply data.Rmd:58]: Ok, I can change/update that.

**Review by cmorganl — APPROVED** [review]

**Pull request merged**